index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>Youtube clone</title>
+
+  <link rel="icon" href="images/favicon.ico" type="image/x-icon">
+
+  <link
+    href="css/tags.css"
+    rel="stylesheet"
+  />
+  <link
+    href="css/classes.css"
+    rel="stylesheet"
+  />  
+  <link
+  href="css/elements.css"
+  rel="stylesheet"
+  />
+  <link
+  href="css/highlight.css"
+  rel="stylesheet"
+  />  
+</head>
+<body>
+  <header>
+    <div id="logoAndMenuDiv">
+      <img class="icon" alt="menu" src="images/ThreeLines.jpg">
+      <img class="logo" alt="YouTube Home" src="images/youtubelogo.jpg">  
+    </div>
+    <div id="searchBarDiv">
+      <input id="searchBarInput" type="text" placeholder="Your search here">
+      <button type="submit"><img class="icon" alt="Search" src="images/SearchButton.jpg"></button>
+    </div>
+    <div id="rightTopIconsDiv">
+      <img class="icon" alt="Create" src="images/VideoIcon.jpg">
+      <img class="icon" alt="YouTube apps" src="images/nineDots.jpg">
+      <img class="icon" alt="Notifications" src="images/bellicon.jpg">
+      <img class="icon" alt="Account" src="images/avtar.jpg">  
+    </div>
+
+  </header>
+  <main >
+    <article class="underVideoContent">
+        <iframe id="iFrame" src="https://www.youtube.com/embed/XYeuvbhKy4I" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+        <h1 id="videoTitleH1">I Will Survive | Gloria Gaynor | funk cover ft. Mario Jose</h1>
+        <img id="viewsLikeShare" alt="Video information and sharing"  src="images/viewsLikesShare.jpg">
+        <img id="channelAndSubscribeImg" alt="Channel and subscribe button" src="images/channelAndSubscribe.jpg">
```

Comment:
It is not allowed to use images, you have to create the structure of the design using HTML and CSS

Reply:
Thank You Haddad for your constructive feedback. We've worked on all the suggested improvements.
Thanks!